extensions/amp-story/1.0/amp-story.js

```diff
@@ -418,6 +419,19 @@ export class AmpStory extends AMP.BaseElement {
     });
   }
 
+  /**
+  * @private
+  */
+  setThemeColor_(){
+    // The theme color should be copied from the story's primary accent color
+    // if possible, with the fall back being default light gray.
+    let meta = document.createElement('meta');
+    const styles = getComputedStyle(document.body);
+    meta.name = "theme-color";
+    meta.content = styles.getPropertyValue('primary-color') || '#F1F3F4';
```

Comment:
nit: can you extract the `#F1F3F4` to a constant (e.g. `DEFAULT_THEME_COLOR`) declared at the top of this file?